art/classifiers/classifier.py

```diff
@@ -275,6 +279,29 @@ def _apply_preprocessing_standardisation(self, x):
 
         return res
 
+    def _apply_postprocessing(self, preds, fit):
```

Comment:
I think we need to add a call to `postprocessor.estimate_gradient`.

Reply:
as per discussion, we dont need this.